### ISSUE TITLE
icinga2: Can't send external Icinga command: 404 No objects found.

### ISSUE THREAD

**Issue by shellscriptcode** [Icinga/icingaweb2#2701]
I am facing this issue while, disabling checks on icingaweb2 , can you please suggest?

icinga2: Can't send external Icinga command: 404 No objects found.

**Comment by dnsmichi** [comment]
Please provide more details on how to reproduce the issue.

- `icinga2 --version`
- Icinga Web 2 version (System -> About -> Screenshot)
- Which command transport is used (I would guess, the REST API)
- Which service is affected (screenshot)
- How does its corresponding Icinga 2 configuration look like (`icinga2 object list --type Service`)
- What exactly does "disabling checks on icingaweb2" mean?

**Comment by lippserd** [comment]
This has been addressed w/ commit 0e63434d904c3177ff14c22c36f5788d850d1a8b. The initial documentation for the permissions was wrong. The Web 2 API user needs at least the following permissions.

```
[ "actions/*", "objects/modify/hosts", "objects/modify/services", "objects/modify/icingaapplication" ]
```
